nextflow-lib/ena-analysis-submitter.nf

```diff
@@ -35,10 +35,10 @@ process ena_analysis_submit {
     mkdir -p ${run_accession}_output/${study_accession}
     cp ${config_yaml} ${run_accession}_output/${study_accession}
 
-    if [ "${study_accession}" = 'PRJEB45555' ]; then
-        analysis_submission.py -t ${test_submission} -o ${run_accession}_output/${study_accession} -p PRJEB57995 -s ${sample_accession} -r ${run_accession} -f ${output_bam},${output_coverage_gz},${output_annot_vcf_gz} -a PATHOGEN_ANALYSIS -au \${webin_id} -ap \${webin_password}
-        analysis_submission.py -t ${test_submission} -o ${run_accession}_output/${study_accession} -p PRJEB57993 -s ${sample_accession} -r ${run_accession} -f ${filtered_vcf_gz} -a COVID19_FILTERED_VCF -au \${webin_id} -ap \${webin_password}
-        analysis_submission.py -t ${test_submission} -o ${run_accession}_output/${study_accession} -p PRJEB57992 -s ${sample_accession} -r ${run_accession} -f ${consensus_fasta_gz} -a COVID19_CONSENSUS -au \${webin_id} -ap \${webin_password}
+    if [ "${study_accession}" = 'PRJEB59442' ]; then
```

Comment:
Please revert it to the root project, which is the indicator that it is public COVID data here. Everything else is correct.

Please also change set.achived.sh by adding a new parent next time. This is not your fault. I forgot to tell you twice so I updated it twice myself. I finally learned my lesson.